### ISSUE TITLE
Suggestion: Improve manual

### ISSUE THREAD

**Issue by ghp-dev** [docToolchain/docToolchain#233]

**Comment by rdmueller** [comment]
yes, agreed that you view is more appealing. But are we allowed to use all those icons?
I guess most of theme are not a problem but with MS, EA and Atlassian I am unsure.

What if we create something more like a landing page (a jekyll web page for docToolchain would be great, like the one we did for https://docs-as-co.de .
There we could setup your ideas in an even more appealing way. 

What do you think?

**Comment by gernotstarke** [comment]
we have to somehow investigate if the use of these icons is ok - on the other hand,
we could simply add a catchall-copyright phrase (icons are copyrighted by their respective holders)
and maybe list Microsoft, Atlassian, Adobe etc. 

I'm quite sure that nobody will EVER complain, as we do something POSITIVE with all these tools...

**Comment by gernotstarke** [comment]
didn't know Guido was an artist :-)

**Comment by ghp-dev** [comment]
Most of the icons are stencils / shapes from draw.io.
IMHO this [support article](https://desk.draw.io/support/solutions/articles/16000042494-what-are-the-usage-terms-for-diagrams-produced-by-draw-io-) states that we may use the icons. (But I'm not a lawyer)
EA, ASCIIDOC and REVEAL.JS might still need some form of clearance.

**Comment by ghp-dev** [comment]
> [...] landing page [...] There we could setup your ideas in an even more appealing way.
> 
> What do you think?

Sounds like a splendid idea. Although I do not have a clear picture yet of what we would like to see on that landing page...

**Comment by rdmueller** [comment]
take a look at this page: https://www.docslikecode.com/learn/
this is what I have in mind - a single page where we can use the icons from above for many little boxes which describe the features.
A "read more" link could point to the section in the docs.
Maybe we could re-use your diagram idea as starter where all icons link to the corresponding "little box" on the same page...

**Comment by ghp-dev** [comment]
Yep, sounds good to me. I can provide a (better version) of the diagram

**Icon Clearance**
Most icons are from draw.io. I.e. they are licensed to JGraph Ltd. and AFAIK we can use them.
Three icons still need clearance: EA, ASCIIDOC, REVEAL.JS
Would you take care of that part?

I assume you want to use
https://docs-as-co.de/

If that's ok with you, I'll be glad to help with providing content for the site.

**Comment by rdmueller** [comment]
> Three icons still need clearance: EA, ASCIIDOC, REVEAL.JS
> Would you take care of that part?

yes, I will do so...

**Comment by rdmueller** [comment]
> I assume you want to use
> https://docs-as-co.de/

hm. I think we should keep the website for the tools ( htmlSanityCheck, doctoolchain) separate from the approach (docs as code)

but all websites should link to each other...

what do you think?

so we would have to setup a new netlify site. A repository already exists for the docs...
Maybe we should use antora this time instead of jekyll?

that wouls be an html landing page together with antora generated docs...

**Comment by ghp-dev** [comment]
I will support which ever tool is chosen. Haven't worked with any of the two yet...
